src/Components/ActivationKeys/ActivationKeys.js

```diff
@@ -45,6 +49,10 @@ const ActivationKeys = () => {
         title: 'Delete',
         onClick: () => handleDeleteActivationKeyModalToggle(activationKeyName),
       },
+      {
+        title: 'Edit',
```

Comment:
i always say the wireframes are just guides, but in this case i think they should be followed...Edit should come before Delete.  users are more likely to want to edit a key instead of delete it.

Reply:
other than this order, this LGTM!